WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -830,9 +830,13 @@ public void onStart() {
 
     @Override
     public void setTitle(@NonNull final String title) {
-        mToolbarTitle = title;
-        if (mToolbar != null) {
-            mToolbar.setTitle(title);
+        if (isAdded()) {
+            if (title != null) {
```

Comment:
I think we'd still like to display `getString(R.string.wordpress)` when title is null so, may I suggest we use this code instead of the `if (title != null)` structure:

```
mToolbarTitle = (title == null || title.isEmpty()) ? getString(R.string.wordpress) : title;
```

so to basically include the null check in the ternary operator conditional. What do you think @manisha-A ?

Reply:
Yes, that looks cleaner.